ledger/db/src/lib.rs

```diff
@@ -125,9 +125,6 @@ pub struct LedgerDB {
     /// Key Images by Block
     key_images_by_block: Database,
 
-    /// Metadata - stores metadata information about the database.
-    metadata_store: MetadataStore<LedgerDbMetadataStoreSettings>,
```

Comment:
What was this thing? It's totally unused?

Reply:
It was only used in `new()`, so does not need to be a member